components/rmrk/Gallery/History.vue

```diff
@@ -244,10 +286,16 @@ export default class History extends mixins(ChainMixin, KeyboardEventsMixin) {
   }
 }
 </script>
-<style>
+<style lang="scss">
 .short-identity__table {
   max-width: 50em;
   overflow: hidden;
   text-overflow: ellipsis;
 }
+.box {
+  .table-nav {
+    display: flex;
+    justify-content: space-between;
+  }
+}
```

Comment:
This could be replaced by [Bulma flexbox helpers](https://bulma.io/documentation/helpers/flexbox-helpers/), you could use class `is-flex is-justify-content-space-between`